tests/functional/issues-auth.js

```diff
@@ -53,7 +53,28 @@ define(
           .then(function(text) {
             assert.equal(text, "Close Issue", "Button says Close not Reopen");
           });
+      },
+
+      "Pressing 'l' opens the label editor box": function () {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/70"),
+          ".wc-Issue-commentSubmit",
+          true
+        )
+          .findByCssSelector(".wc-Issue-labels")
+          .click()
+          .type("l")
+          .end()
+          .setFindTimeout(0)
```

Comment:
I think this has the potential to be racy. Can we remove it?  

(It should fall back to the default that `FunctionalHelpers.openPage` uses (`config.wc.pageLoadTimeout`).